bin/test

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+main() {
+    # parse the arguments into variables
+    declare build_dir="${1:-}"
+
+    # check if composer tests are available
+    if composer run-script -l 2>/dev/null | grep -q '^\s*test\b'; then
+        # cd into the build directory and run the tests via composer
+        cd "$build_dir" && composer test
```

Comment:
You could `cd ${1:-}` here directly, but see above

Reply:
You probably want `composer run-script --timeout=0 test`

Reply:
I think the `cd` is correct, as noted above (and [here](https://github.com/heroku/heroku-buildpack-nodejs/blob/master/bin/test#L5)).